vm-setup/roles/v1aX_integration_test/templates/clusterctl_env_template_centos.rc

```diff
@@ -101,25 +100,6 @@ export CTLPLANE_KUBEADM_EXTRA_CONFIG="
               smtp_server localhost
               smtp_connect_timeout 30
           }
-          vrrp_instance VI_1 {
-              state MASTER
-              interface eth1
-              virtual_router_id 1
-              priority 101
-              advert_int 1
-              virtual_ipaddress {
-                  {{ CLUSTER_APIENDPOINT_HOST }}
-              }
-          }
```

Comment:
Why is part of the keepalived config removed ?

Reply:
also, be careful with the interface name here

Reply:
> also, be careful with the interface name here

What does it mean, should not it be `eth1`?

Reply:
Done, thanks.